### PR TITLE
Kh/dr/#10 visitor views project

### LINKED ISSUE #10: Visitor Story: Project show page without login
As a visitor when I visit /projects/:username/:project_name
I should be able to view that project
And I should see a link to support the project
And I should see how many supporters of the project there is
And I should see how many days are left in the project
And I should see how much money is currently pledged, in the form of a progress bar

### PR DESCRIPTION
**Pull request by kheppenstall**
Worked with @drod1000 

-Remove pledged from project table and make a pledged method
-Create project serializer
-Feature test for visitor viewing one project
-Project service and associated tests
-Kickstarter Project model and associated tests

Closes #10

@Dpalazzari @jdconrad89 @Robbie-Smith Lets all review this one!

When you pull this one down then be sure to add this line to your application.yml file.

`project_slug: 'your project slug goes here'`

### CODE REVIEW

File: app/services/project_service.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class ProjectService
+
+  def self.conn
+    Faraday.new(url: "http://localhost:3000/api/v1/projects/") do |faraday|
+      faraday.adapter  Faraday.default_adapter
+    end
```
Comment (Robbie-Smith) [inline app/services/project_service.rb]: On line 5 I think you are missing an '=' sign
faraday.adapter  Faraday.default_adapter

Comment (kheppenstall) [inline app/services/project_service.rb]: @Robbie-Smith I think that is actually how it is recommended to do it in the Faraday docs.

http://www.rubydoc.info/gems/faraday/0.9.2

Comment (Robbie-Smith) [inline app/services/project_service.rb]: Ah yea, you're right. We'll have to investigate the TravisCI more tomorrow then.

File: spec/features/visitor/views_project_show_page_spec.rb [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-    end
 
-    visit "/projects/#{project.slug}"
+  before do
+    VCR.insert_cassette 'visitor_views_project', record: :new_episodes
+  end
+
+  after do
+    VCR.eject_cassette
+  end
```
Comment (Dpalazzari) [inline spec/features/visitor/views_project_show_page_spec.rb]: What is the purpose of this code?

**Comment by Dpalazzari** [comment]
👍

**Pull request merged**